moveit_ros/moveit_servo/CMakeLists.txt

```diff
@@ -52,34 +56,48 @@ include_directories(
   ${Boost_INCLUDE_DIR}
 )
 
-#########################################
-## A library providing a C++ interface ##
-#########################################
+#########################################################
+## Library to process realtime twist or joint commands ##
+#########################################################
 
-add_library(${LIBRARY_NAME} SHARED
+# This library provides an interface for sending realtime twist or joint commands to a robot
+add_library(${SERVO_LIB_NAME} STATIC
```

Comment:
:-1: We should stay with `SHARED` libraries for all public interfaces (so literally all...) in MoveIt.

In ROS `SHARED` is the default as well (which users *can* switch if they really want to), so just drop the specifier altogether. I'm still puzzled why that default apparently changed in ROS2/ament.